cmd/main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 	}
 
 	loader := manifestreader.NewManifestLoader(f)
-	invFactory := inventory.ClusterClientFactory{StatusPolicy: inventory.StatusPolicyNone}
```

Comment:
As I'm looking at the [client](https://github.com/kubernetes-sigs/cli-utils/blob/b6f5dbece92f7053344b91b99b62523176e5a734/pkg/inventory/inventory-client.go#L61-L89) it seems pretty versatile, allowing to work with all the resources (especially that it [embeds dynamic client](https://github.com/kubernetes-sigs/cli-utils/blob/b6f5dbece92f7053344b91b99b62523176e5a734/pkg/inventory/inventory-client.go#L67)). So imo, its current name is pretty accurate.